bigquery/biglake/bigquery_create_object_table/main.tf

```diff
@@ -74,9 +79,8 @@ resource "google_bigquery_table" "default" {
   # Interval literal: https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#interval_literals
   # max_staleness = "0-0 0 10:0:0"
```

Comment:
Suggest dropping lines 77 - 80. If critical to keep, should change the metadata_cache_mode so that we can incorporate.